wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/ListContent/ConversationListAccessoryView.swift

```diff
@@ -93,6 +96,17 @@ final class ConversationListAccessoryView: UIView {
         updateForIcon()
     }
 
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+        guard previousTraitCollection?.userInterfaceStyle != traitCollection.userInterfaceStyle else { return }
+        if icon == .silenced {
+            // We need to call this method here because the background, the border color
+            // of the icon when switching from dark to light mode
+            // or vice versa can be updated only inside traitCollectionDidChange.
+            configureSilencedNotificationsIcon()
+        }
```

Comment:
Couldn't you just call `updateForIcon()`?

Reply:
I can. Updated the code.